client/dashboard/data/staging-sites.ts

```diff
@@ -0,0 +1,9 @@
+import wpcom from 'calypso/lib/wp';
```

Comment:
Can we call the file `data/site-staging-site.ts`? In this layer, we tend to match the filename with the actual API paths.